handlers/handlers.go

```diff
@@ -89,14 +111,92 @@ func HandleSessionRequest(
 			panic(err)
 		}
 
+		// Create token as a cookie and set it
+		{
+			userId, err := userservice.GetUserIdFromEmailAddress(loginForm.EmailAddress)
+			if err != nil {
+				panic(err)
+			}
+
+			token, err := createTokenAsString(userId, oneWeekInMinutes)
```

Comment:
why do we need to specify one week in two places?

Reply:
one is for the token the other is for the cookie. I gave them the same expiration to prevent sheer anarchy.

Reply:
Feels a bit like overkill, but it would be nice if there was a single variable we could change that would change the expiration time.

Reply:
ask and thee shall receive